iphone/Classes/ContactsModule.m

```diff
@@ -899,14 +899,23 @@ -(void)peoplePickerNavigationControllerDidCancel:(ABPeoplePickerNavigationContro
 }
 
 //Deprecated in iOS 8
--(BOOL)peoplePickerNavigationController:(ABPeoplePickerNavigationController *)peoplePicker shouldContinueAfterSelectingPerson:(ABRecordRef)person
+-(BOOL)peoplePickerNavigationController:(ABPeoplePickerNavigationController *)peoplePicker shouldContinueAfterSelectingPerson:(ABRecordRef)selectedPerson
 {
 	if (selectedPersonCallback) {
-		ABRecordID id_ = ABRecordGetRecordID(person);
-		TiContactsPerson* person = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext] recordId:id_ module:self] autorelease];
+		TiContactsPerson* person = nil;
+		if ([TiUtils isIOS8OrGreater] && (ABAddressBookGetAuthorizationStatus() != kABAuthorizationStatusAuthorized)) {
+			// In iOS 8 selected contact is returned without requiring user permission. But we cannot query metadata like recordid.
+			person = [[[TiContactsPerson alloc] _initWithPageContext:[self executionContext]
```

Comment:
Looks like objective-c style method formatting is the issue. Changed it to single line. Please check.